src/scripts/items/BattleItem.ts

```diff
@@ -20,4 +20,15 @@ ItemList['xClick']          = new BattleItem(GameConstants.BattleItemType.xClick
 ItemList['Lucky_egg']       = new BattleItem(GameConstants.BattleItemType.Lucky_egg, '+50% Bonus to experience gained', 800, undefined, 'Lucky Egg');
 ItemList['Token_collector'] = new BattleItem(GameConstants.BattleItemType.Token_collector, '+50% Bonus to dungeon tokens gained', 1000, undefined, 'Token Collector');
 ItemList['Item_magnet']     = new BattleItem(GameConstants.BattleItemType.Item_magnet, '+50% Chance of gaining an extra item', 1500, undefined, 'Item Magnet');
-ItemList['Lucky_incense']   = new BattleItem(GameConstants.BattleItemType.Lucky_incense, '+50% Bonus to money gained', 2000, undefined, 'Lucky Incense');
\ No newline at end of file
+ItemList['Lucky_incense']   = new BattleItem(GameConstants.BattleItemType.Lucky_incense, '+50% Bonus to money gained', 2000, undefined, 'Lucky Incense');
+
+namespace BattleItems {
+    export enum BattleItem {
+        'xAttack' = 0,
+        'xClick',
+        'Lucky_egg',
+        'Token_collector',
+        'Item_magnet',
+        'Lucky_incense',
+    }
+}
```

Comment:
I don't think we should do this, as it just creates more places that need to be edited when we add a new battle item.
If we need a numeric enum, we should probably modify `GameConstants.BattleItemType` to fit our needs.

Reply:
Fixed, though I couldn't use SeededRand.fromEnum, since the enum values for BattleItemType aren't numbers.